03_letters/CameronSchafer/letters.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
+func letters(s string) map[rune]int {
+	tMap := make(map[rune]int)
+
+	for i := range s {
+		tMap[rune(s[i])]++
+	}
+
+	return tMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	tSlice := []int{}
```

Comment:
the more `go` way of delaring empty slices would be
`var tSlice []int`
or if we're going for optimisation
`tSlice := make([]int, 0, len(m))` this will preallocate only the needed memory to tSlice

Reply:
This file was removed because wrong lab